dataland-external-storage/src/main/kotlin/org/dataland/datalandexternalstorage/services/DatabaseConnection.kt

```diff
@@ -1,3 +1,4 @@
+import com.fasterxml.jackson.databind.ObjectMapper
 import org.slf4j.LoggerFactory
 import java.sql.Connection
 import java.sql.DriverManager
```

Comment:
Gab es einen bestimmten Grund, weshalb das hier als `object` umgesetzt wurde und nicht als Spring @Component?

Denn im Grunde ist das ein DatabaseConnector oder DatabaseOperator, der für die Kommunikation mit der Datenbank verantwortlich ist.

Reply:
Ok aber es wird nur vom EurodatDataStore benutzt und da hast du erst 5 Autowires.
6 sind glaube ich erlaubt von detekt. D.h. du hättest eigentlich keine Probleme.
Dann wärst du konsistent mit unserem restlichen Code

Reply:
Dann würde ich es auch als Service betrachten und "EurodatDatabaseConnector" nennen, da im Code sowieso EuroDaT mehrfach genannt wird.

Reply:
Aber was ist daneben der Vorteil? Vorteil von einer class ist dass es mehrere instances geben kann, beim object nur eine. Brauchen wir denn mehr als eine instance?

Reply:
Es liegt ja sogar bereits im "services" Ordner

Reply:
Lass uns das auch in dem Call besprechen

Reply:
Von unseren services gibt es auch nur eine instance, weil Spring eine @Component per default als Singleton baut.
Es ist also auch dann nur eine instance da.

Ich habe kein starkes Argument dafür, das als @Component zu machen, aber ich finde auch keinen guten Grund dafür, wieso wir bei diesem File auf diese Spring-Funktionalität verzichten sollten, wenn wir eine Spring-Boot-App schreiben. 

Können nochmal darüber sprechen.